Spigot-API-Patches/0300-ItemStack-repair-check-API.patch

```diff
@@ -0,0 +1,60 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jake Potrebic <jake.m.potrebic@gmail.com>
+Date: Sat, 15 May 2021 22:10:50 -0700
+Subject: [PATCH] ItemStack repair check API
+
+
+diff --git a/src/main/java/org/bukkit/UnsafeValues.java b/src/main/java/org/bukkit/UnsafeValues.java
+index 6dbd520182b1e7713a68baad09b7f613424ef619..a7e72154dc8b16e28514baffdf91fbda93e0def0 100644
+--- a/src/main/java/org/bukkit/UnsafeValues.java
++++ b/src/main/java/org/bukkit/UnsafeValues.java
+@@ -155,6 +155,17 @@ public interface UnsafeValues {
+      */
+     public io.papermc.paper.inventory.ItemRarity getItemStackRarity(ItemStack itemStack);
+ 
++    /**
++     * Checks if an itemstack can be repaired with another itemstack.
++     * <p>
++     * <b>Both Itemstacks' materials must be items ({@link Material#isItem()}).</b>
```

Comment:
Why? I don't see a reason for this, as blocks obviously cannot be repaired, so should rather always return `false`.

Reply:
Yeah, ok. removes the need for the preconditions, can just return false